build-system/tasks/check-owners.js

```diff
@@ -72,4 +119,5 @@ module.exports = {
 checkOwners.description = 'Checks all OWNERS files in the repo for correctness';
 checkOwners.flags = {
   'files': '  Checks only the specified OWNERS files',
+  'validate': '  Validates the OWNERS files with the owners bot API',
```

Comment:
I don’t see this flag being used anywhere, so it can be removed.

Also, `local_changes` is missing from this list.

Reply:
That wasn't supposed to get caught in the commit, good catch!

Reply:
Removed.

Reply:
I still don’t see `local_changes`

Reply:
Added :+1: